### ISSUE TITLE
Loudness normalization

### ISSUE THREAD

**Issue by unfa** [x42/sound-gambit#4]
I wonder - could sound-gambit perform loudness normalization?
As in - making sure that a specified LUFS Integrated levels is achieved in the output?

I tried doing this with sox - but it's fixed-point so 32-bit float WAVs that cross 0dBFS will get clipped anyway.
I tried ffmpeg but got stuck due to the complexity, and I am also not sure if it can cleanly work on 32-bit float files.

**Comment by x42** [comment]
That is what https://github.com/ElizabethHarmon/ebu-norm does. It uses sound-gambit under the hood.

Normalizing loudness is a two step process: first analyze and then apply gain and limit.

**Comment by ghost** [comment]
Just an FYI, I did the unthinkable and changed the location: https://github.com/ElizabethHarmon/ebu-norm-tools

**Comment by dromer** [comment]
Hmm, and now entirely removed from the internet?

I have a backup of `ebu-norm` here -> https://github.com/Wasted-Audio/ebu-norm

**Comment by chmaha** [comment]
No, I temporarily quit github for reasons I won't go into but I put everything back up a good while ago and the ebu-norm-tools repo is here: https://github.com/chmaha/ebu-norm-tools (and not going anywhere). Apologies for any inconvenience.
